plugins/SimulationView/SimulationView.py

```diff
@@ -360,6 +364,9 @@ def setPath(self, value: float) -> None:
         if self._current_path_num != value:
             self._current_path_num = min(max(value, 0), self._max_paths)
             self._minimum_path_num = min(self._minimum_path_num, self._current_path_num)
+            # update _current time when the path is changed by user
+            if self.getCurrentLayer() in self._cumulative_line_duration  and self._current_path_num < self._max_paths and round(self._current_path_num)== self._current_path_num:
+                self._current_time = self._cumulative_line_duration[self.getCurrentLayer()][int(self._current_path_num)]
```

Comment:
Why only do this when `self.getCurrentLayer() in self._cumulative_line_duration` I would argue, if `_cumulative_line_duration`
```suggestion
            comulative_line_duration = self.comunativeLineDuration()
            self._current_time = comulative_line_duration[int(self._current_path_num)]
```

Reply:
fixed in 0de4f612b399cfe8b7a6c8059e9579489bcd739c